data/110/869/570/1/1108695701.geojson

```diff
@@ -10,6 +10,18 @@
     "geom:latitude":33.930031,
     "geom:longitude":76.509741,
     "iso:country":"IN",
+    "label:eng_x_preferred_longname":[
+        "Kargil District"
+    ],
+    "label:eng_x_preferred_placetype":[
+        "district"
+    ],
+    "label:hin_x_preferred_longname":[
+        "\u0915\u093e\u0930\u0917\u093f\u0932 \u091c\u093c\u093f\u0932\u093e \u091c\u093c\u093f\u0932\u093e"
```

Comment:
Looks like the district text must be in the `name`, so check to make sure it's not repeating.